scripts/zones/Quicksand_Caves/IDs.lua

```diff
@@ -117,6 +120,26 @@ zones[xi.zone.QUICKSAND_CAVES] =
         CHAINS_THAT_BIND_US_QM = 17629746,
         TREASURE_COFFER        = 17629747,
         ANTICAN_TAG_QM         = 17629769,
+        QM_AMK_1               = 17629668,
```

Comment:
These are still raw IDs, you need to be using `GetFirstID` or `GetTableOfIDs` to pull these from the db at load time - which makes them more shift-proof

Reply:
That is really cool.